LTTngCds/LTTngCds.csproj

```diff
@@ -33,6 +33,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\CtfPlayback\CtfPlayback.csproj" />
```

Comment:
Please also increment the versions of the plugins by .1 for all of these in the .csproj and the plugin.json